src/Strategy.js

```diff
@@ -0,0 +1,73 @@
+//@flow
```

Comment:
Can we rename `ElementPropSpec` to `Strategy` than, since it will not define how to resolve only `Element` but also `style`. We can rename Strategy to `Resolver` or smth like that